ui/pages/index.js

```diff
@@ -1,7 +1,14 @@
 import { useRouter } from 'next/router'
+import { useEffect } from 'react'
 
 export default function Index() {
-  useRouter().replace('/destinations')
+  const router = useRouter()
```

Comment:
Removing the sign-up API check exposed this issue where the router could be called during server-side rendering. Putting the re-route in the use effect ensure this is only called client-side.